### DISCUSSION TITLE
unable to build docker image on ubuntu on RPi

### DISCUSSION THREAD

**Discussion by sandeep-gh** [termux/termux-packages#7565] [Q&A] [answered]
Hello,

 I am trying to build python3.10 on termux and for that was creating Docker build environment on ubuntu which is running on RPi4. 
However, the command `docker build -t termux/package-builder .` fails with error: 
```E: Failed to fetch http://ports.ubuntu.com/ubuntu-ports/dists/focal/restricted/binary-i386/Packages  404  Not Found [IP: 91.189.91.39 80]```. 

None of the suggestion mentioned on the web seems to be applicable here. Is there something I need to change in setup-ubuntu.sh?

Thanks
-S

**Comment by ghost** [comment, marked as answer]
Sorry, Docker image can be built only for x86_64. Reason is mainly for Android NDK and SDK. However rest expects x86_64 machine anyway.

**Reply by sandeep-gh** [reply]
Got it.  Thanks for the note. Will try on x86 machine then.
